iosApp/iosAppTests/Pages/Map/HomeMapViewTest.swift

```diff
@@ -19,7 +19,7 @@ final class HomeMapViewTest: XCTestCase {
         executionTimeAllowance = 60
     }
 
-    func testAppears() throws {
+    @MainActor func testAppears() throws {
```

Comment:
Is there a reason related to this change that we need this macro now?

Reply:
I'm not sure. It might have been broken for a bit and we didn't notice. When running just the tests it doesn't seem to matter but SwiftUI previews seem to also compile the test suite and throw an error if this macro isn't there (even though it's not actually running the tests).